src/setting-password-reset.php

```diff
@@ -129,57 +111,55 @@
-                                                        <div class="col-md-12 pb-3">
-                                                            <p class="fw-bold fs-3 text-success spacing-4">Password Updated</p>
-                                                            <p class="fw-medium spacing-3">Your password has been successfully updated! Please log in again.</p>
-                                                            <button class="button-check text-white py-2 px-2" id="Home" style="background-color: #27AE60; font-size: 15px; width: 200px;">
-                                                                <a class="custom-link" href="../src/setting-email-update.php"><b>Go to Landing Page</b></a>
-                                                            </button>
-                                                        </div>
-                                                    </div>
-                                                    <p>Redirecting to landing page in <b>10</b> seconds...</p>
-                                                </div>
-                                            </div>
-                                        </div>
+                                <div class="row">
+                                    <div class="col-md-12 pb-3">
+                                        <p class="fw-bold fs-3 text-success spacing-4">Password Updated</p>
+                                        <p class="fw-medium spacing-3">Your password has been successfully updated! Please log in again.</p>
+                                        <button class="button-check main-bg-color text-white py-2 px-4 custom-link" onclick="window.location.href='includes/voter-logout.php'"><b>Back to Landing Page</b></button>
                                     </div>
                                 </div>
-
-                                <script>
-                                    $(document).ready(function() {
-                                        $('#SCO-login-button').click(function(event) {
-                                            event.preventDefault(); // Prevent form submission
-                                            $('#successEmailModal').modal('show');
-                                        });
-                                    });
-                                </script>
+                                <p class="timer">Redirecting to landing page in <strong>10</strong> seconds...</p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <script>
+                $(document).ready(function() {
+                    $('#submit').click(function(event) {
+                        event.preventDefault(); // Prevent form submission
+                        $('#successEmailModal').modal('show');
+                    });
+                });
+            </script>
```

Comment:
What is the purpose of this code? From what I understood, this is supposed to show a success email  modal after successful password changed. But there's no API/endpoint calls to validate the submitted password to the server-side?

Also, as far as I can remember, this page serves as a medium for the user to change his password?